mos-platform/snes/vectors.s

```diff
@@ -0,0 +1,301 @@
+; SNES Interrupt Vector Handlers for LLVM-MOS (6502 compatible)
+; Based on vectors.asm reference from wdc816cc
```

Comment:
We can't really "base" code on WDC's, unfortunately. I assume the licenses are not compatible.

However, what little potentially copyrightable material I see in this file requires a rework anyway...